### PR TITLE
add support for left hand coordinate system for POI feature

### PR DESCRIPTION
**Pull request by runpiw**
## Description
POI feature is not working properly if the babylon scene is not set to use right hand coordinate system.
GLTF is in right hand coordinate system while babylon scene by default is using left hand coordinate system so that babylon GLTF importer is applying some transform to the root node of imported character which breaks our POI feature angle calculation. 
This change invert one of the angle calculation in POI feature if the babylon scene is using left hand coordinate system to mitigate the transform impact brought by babylon GLTF importer
## Testing
Manually go through all babylon examples to verify POI feature is working properly.
## Submission Checklist
I confirm that I have...
- [x] removed hard-coded Cognito IDs
- [x] manually smoke-tested the BabylonJS integration tests
- [x] manually smoke-tested the BabylonJS demos
- [x] manually smoke-tested the Three.js integration tests
- [x] manually smoke-tested the Three.js demo

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by JuliaABurch — APPROVED** [review]
It's too bad we don't have any unit tests or integration tests that target the POI feature -- in their absence, would it be worth removing the `useRightHandedSystem` from one of the demos (perhaps the Hello World one)? 

At the very least, those demos should be updated in the sense that there's a comment in the `.js` files' `createScene()` function that says `// IMPORTANT for Sumerian Hosts!` (or something along those lines) that should be removed

**Review by eherozhao — DISMISSED** [review]

**Comment by runpiw** [comment]
> It's too bad we don't have any unit tests or integration tests that target the POI feature -- in their absence, would it be worth removing the `useRightHandedSystem` from one of the demos (perhaps the Hello World one)?
> 
> At the very least, those demos should be updated in the sense that there's a comment in the `.js` files' `createScene()` function that says `// IMPORTANT for Sumerian Hosts!` (or something along those lines) that should be removed



> It's too bad we don't have any unit tests or integration tests that target the POI feature -- in their absence, would it be worth removing the `useRightHandedSystem` from one of the demos (perhaps the Hello World one)?
> 
> At the very least, those demos should be updated in the sense that there's a comment in the `.js` files' `createScene()` function that says `// IMPORTANT for Sumerian Hosts!` (or something along those lines) that should be removed

Removed the useRightHandedSystem setting from Hello World demo and updated comments

File: packages/amazon-sumerian-hosts-babylon/src/Babylon.js/PointOfInterestFeature.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ class PointOfInterestFeature extends CorePointOfInterestFeature {
     return [...obj.getWorldMatrix().m];
   }
 
+  static _sphericalToBlendValue(...args) {
+    let {h, v} = super._sphericalToBlendValue(...args);
+
+    //Revert theta blend value if not using right hand coordinate system
```
[the remark this answers is not in the document]
Comment (runpiw) [reply packages/amazon-sumerian-hosts-babylon/src/Babylon.js/PointOfInterestFeature.js:25]: I refer to the value as theta blend value which i think it works as well

**Review by JuliaABurch — APPROVED** [review]

**Review by eherozhao — APPROVED** [review]

File: packages/demos-babylon/src/helloWorldDemo.js [hunk 2]
```diff
@@ -7,10 +7,9 @@ let host;
 let scene;
 
 async function createScene() {
-  // Create an empty scene. IMPORTANT: Sumerian Hosts require use of the
-  // right-hand coordinate system!
-  scene = new Scene();
-  scene.useRightHandedSystem = true; // IMPORTANT for Sumerian Hosts!
+  // Create an empty scene. Note: Sumerian Hosts work with both
+  // right-hand or left-hand coordinate system for babylon scene
+  scene = new BABYLON.Scene();
```
Comment (prestomation) [inline packages/demos-babylon/src/helloWorldDemo.js:12]: This change broke the demos as we have reverted this line to a previous change. The warnings above should have been a pretty good hint!

**Pull request merged**